2024-04-18 10:32:27,039 - Log.py[line:32] - INFO: Serial port connection succeeded
2024-04-18 10:32:27,568 - Log.py[line:32] - INFO: Successfully connected to the server
2024-04-18 10:32:28,544 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:29,715 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:30,707 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:32,885 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:34,025 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:35,034 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:35,979 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:37,021 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:38,094 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:39,146 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:40,262 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:41,237 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:42,230 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:43,215 - Log.py[line:36] - ERROR: Invalid version: ''
2024-04-18 10:32:44,395 - Log.py[line:36] - ERROR: Invalid version: ''
